swarm/network/simulation/kademlia_test.go

```diff
@@ -63,4 +85,47 @@ func TestWaitTillHealthy(t *testing.T) {
 			t.Fatal(err)
 		}
 	}
+
+	// now create a snapshot of this network
+	snap, err := sim.Net.Snapshot()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// create a control simulation
+	controlSim := New(simServiceMap)
```

Comment:
this sim should run without discovery no??

Reply:
To be correct strictly, yes